.chloggen/propagate-global-scrape-protocols.yaml

```diff
@@ -5,12 +5,12 @@ change_type: enhancement
 component: target allocator
 
 # A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
-note: introduces the global field in the TA config to allow for setting scrape protocols
+note: Rollback 3187
 
 # One or more tracking issues related to the change
-issues: [3160]
+issues: [3242]
```

Comment:
I think it would make sense to create a new changelog entry for the rollback.

Reply:
Thanks, I'm unfamiliar with the process. I think I've done it correctly now, but let me know if I missed something.